openshift/app.dc.yaml

```diff
@@ -132,6 +132,17 @@ objects:
                 limits:
                   cpu: "${LOGGING_CPU_LIMIT}"
                   memory: "${LOGGING_MEMORY_LIMIT}"
+              env:
+                - name: AWS_ACCESS_KEY_ID
+                  valueFrom:
+                    secretKeyRef:
+                      key: username
+                      name: "${APP_NAME}-aws-kinesis-secret"
+                - name: AWS_SECRET_ACCESS_KEY
+                  valueFrom:
+                    secretKeyRef:
+                      key: password
+                      name: "${APP_NAME}-aws-kinesis-secret"
```

Comment:
May be worth considering renaming these env variables to specifically `KINESIS_*` as these access keys are specific to that service only.

Reply:
these env variables have to be named this way so that the aws kinesis fluent-bit output plugin can find them. ref: https://github.com/aws/amazon-kinesis-streams-for-fluent-bit#credentials